tests/pipelines/test_stale_monitor.py

```diff
@@ -0,0 +1,112 @@
+import rethinkdb as r
+from bigchaindb import Bigchain
+from bigchaindb.pipelines import stale
+from multipipes import Pipe, Pipeline
+from unittest.mock import patch
+from bigchaindb import config_utils
+import time
+import os
+
+
+def test_get_stale(b, user_vk):
+    tx = b.create_transaction(b.me, user_vk, None, 'CREATE')
+    tx = b.sign_transaction(tx, b.me_private)
+    b.write_transaction(tx, durability='hard')
+
+    stm = stale.StaleTransactionMonitor(timeout=0.001,
+                                        backlog_reassign_delay=0.001)
+    tx_stale = stm.check_transactions()
+
+    for _tx in tx_stale:
+        assert tx == _tx
+
+
+def test_reassign_transactions(b, user_vk):
```

Comment:
To avoid depending on the system clock you can:
- put a `monkeypatch.setattr(time, 'time', lambda: 1)` before L:26
- change `timeout=0.001` to `timeout=0`, and `backlog_reassign_delay=1`
- put a `monkeypatch.setattr(time, 'time', lambda: 10)` before L:32
- [and do the same with the rest of the test]

(if you are using `r.now` then you should monkeypatch something different)